moped-editor/src/App.js

```diff
@@ -80,11 +81,13 @@ const App = () => {
         <LocalizationProvider dateAdapter={AdapterDateFns}>
           <ThemeProvider theme={theme}>
             <GlobalStyles />
-            <ProjectListViewQueryContext.Provider
-              value={{ listViewQuery, setListViewQuery }}
-            >
-              {routing}
-            </ProjectListViewQueryContext.Provider>
+            <ActivityMetrics>
+              <ProjectListViewQueryContext.Provider
+                value={{ listViewQuery, setListViewQuery }}
+              >
+                {routing}
+              </ProjectListViewQueryContext.Provider>
+            </ActivityMetrics>
```

Comment:
keeping it simple here and just wrapping the entire app underneath this. if we wanted to get fancy, we could push this into specific pages/components and add arguments to the event handler function to log them.